### ISSUE TITLE
Spelling Errors not Announced in Google Docs

### ISSUE THREAD

**Issue by frastlin** [nvaccess/nvda#9043]
### Steps to reproduce:
1. Open a Google Docs file
2. Type: flred
3. hit space
4. hit down arrow and up arrow

### Actual behavior:

NVDA makes no buzz and does not announce spelling error.

### Expected behavior:

NVDA makes a buzz sound when space is pressed and says "spelling error" when NVDA reads the word after up arrow is pressed.

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2018.3.2
#### Windows version:
Windows 10 64 Bit
#### Name and version of other software in use when reproducing the issue:
Google Docs
Chrome Version 70.0.3538.110 (Official Build) (64-bit)
Firefox 63.0.3 (64-bit)

#### Other information about your system:
Google docs is in Braille mode, but for the most part the same happens if Braille mode is not enabled. The only exception is when the misspelled word is on its own line and you have Braille mode disabled and you arrow line by line over the misspelling.
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

**Comment by Adriani90** [comment]
Closing as duplicate of #8155. Thanks for reporting. Please continue the discussion on issue #8155
